validation/tests/fixtures/prescriptor/bad_dates_submission.csv

```diff
@@ -0,0 +1,15 @@
+CountryName,RegionName,Date,C1_School closing,C2_Workplace closing,C3_Cancel public events,C4_Restrictions on gatherings,C5_Close public transport,C6_Stay at home requirements,C7_Restrictions on internal movement,C8_International travel controls,H1_Public information campaigns,H2_Testing policy,H3_Contact tracing,H6_Facial Coverings,PrescriptionIndex
```

Comment:
The old `fixtures` files have moved to a `predictor` subfolder. These new files are in a `prescriptor` subfolder